Tests/Marketplace/zip_packs.py

```diff
@@ -244,12 +288,20 @@ def main():
     zipped_packs = []
     success = True
     try:
-        zipped_packs = download_packs_from_gcp(storage_bucket, gcp_path, zip_path, circle_build, branch_name)
+        download_packs_from_gcp(storage_bucket_name, gcp_path, zip_path, circle_build, branch_name)
     except Exception:
         logging.exception('Failed downloading packs')
         success = False
 
-    if remove_test_playbooks:
+    try:
+        zipped_packs = get_zipped_packs_names(zip_path, gcp_path, circle_build)
+    except Exception:
+        logging.exception('No zip files were found')
+        success = False
+
+    copy_to_other_dir(zipped_packs)
+
+    if zipped_packs and remove_test_playbooks:
```

Comment:
Currently, if `zipped_packs` is empty the script has already ended.

Reply:
As mentioned by @liorblob above. If zipped packs is empty we still need to call the cleanup function in the end of the script.

Reply:
In the `get_zipped_packs_names` function you have the following code:
`    if not zipped_packs:
        sys.exit(1)`
So it's not going to get to line 304 in this case.

Reply:
You're right. I changed it now so there's only sys.exit(1) after cleanup was called and only if success is false